common/djangoapps/enrollment/views.py

```diff
@@ -276,6 +280,36 @@ def get(self, request, course_id=None):
             )
 
 
+class EnrollmentCourseRosterView(APIView, ApiKeyPermissionMixIn):
+    @method_decorator(ensure_csrf_cookie_cross_domain)
+    def get(self, request, course_id=None):
+        if not self.has_api_key_permissions(request):
+            return Response(
+                status=status.HTTP_401_UNAUTHORIZED,
+                data={
+                    "message": u"The server that sent this request does not \
+                    have a valid API key",
+                }
+            )
+        username = request.GET.get('user', request.user.username)
```

Comment:
confirm what this is doing,
probably just want to use request.user.username
username = request.user.username